### ISSUE TITLE
Request: Add back the ability to equip "Loot" inventory items.

### ISSUE THREAD

**Issue by GambetTV** [foundryvtt/dnd5e#2204]
You can equip literally everything else. There's no reason to take away features that already existed like this. It just kills modules that utilize it, and basically limits what we can do with our games. Maybe I want to know if a player is holding his tinderbox. There are so many items that don't have a mechanical reason to be classified as Equipment or Weapons that we still might have good reasons for wanting to equip in game. The module Variant Encumbrance, among others, utilized equipping items to reduce carry weight. Removing features like this hinders my ability to homebrew.

**Comment by Aize01** [comment]
Yep, very essential for my campaign as well, since we play heavily on inventory management (its a scavenging apocalyptic campaign), please bring this back.

**Comment by Fyorl** [comment]
Seemingly a deliberate decision per #1992 but we've since moved it back to consumables and containers. I do not recall the original rationale for this so I will tag it as a feature request to consider.
